test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -224,13 +174,127 @@ public void setValueAt(Object aValue, int row, int column) {
     }
 
     public static void main(String[] args) throws Exception {
-        SwingUtilities.invokeAndWait(() -> {
-            try {
-                initTest();
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        });
-        passFailJFrame.awaitAndCheck();
+        String INSTRUCTIONS = """
+                Instructions to Test:
+                1. Refer the below keyboard navigation specs
+                 (referenced from bug report 4112270).
+                2. Check all combinations of navigational keys in all four modes
+                  shift and control verifying each change to the selection against
+                  the spec. If it does, press "pass", otherwise press "fail".
+
+                """;
+
+        INSTRUCTIONS += getOSSpecificInstructions();
+        PassFailJFrame.builder()
+                .instructions(INSTRUCTIONS)
+                .rows(11)
```

Comment:
```suggestion
                .rows((int) INSTRUCTIONS.lines().count() + 2)
```

This is what we used for our tests. I default to using this now myself.

Reply:
This doesn't work here since the Instruction rows are quite more.

Reply:
@TejeshR13 Rows can be increased so as to reduce scrolling.

Reply:
Increased to 30 rows, guess this would be optimal size.